library/pk_wrap.c

```diff
@@ -1602,6 +1602,32 @@ const mbedtls_pk_info_t mbedtls_pk_ecdsa_opaque_info = {
     NULL, /* debug - could be done later, or even left NULL */
 };
 
+#if defined(MBEDTLS_RSA_C)
```

Comment:
I don't think this should depend on `MBEDTLS_RSA_C`, as support for RSA decryption in PSA might be provided by a driver rather than by our own `rsa.c`. This is already guarded by `MBEDTLS_USE_PSA_CRYPTO` (which implies `MBEDTLS_PSA_CRYPTO_C` and I think it's enough.

Reply:
Ok, now there's a dependency issue with `mbedtls_pk_error_from_psa_rsa` when I remove `MBEDTLS_RSA_C`

Reply:
@mpg is it ok to replace`MBEDTLS_RSA_C` guard with `PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY` so `mbedtls_pk_error_from_psa_rsa()` can be used ?